src/aics-image-viewer/assets/styles/variables.css

```diff
@@ -1,14 +1,16 @@
-$app-height: 100%;
-$dark-gray: #313131;
-$purple: #8048f3;
-$highlight-purple: #8950d9;
-$light-gray: #6e6e6e;
-$bright-green: #b2d030;
-$background-gray: #fafafa;
-$header-gray: #4b4b4b;
-$background-dark-gray: #131313;
+/* TODO: Replace all of these with CSS variables (e.g. var(--color-text-body) and remove PostCSS from the project.*/
+
+/* $app-height: 100%; */
+/* $dark-gray: #313131; */
+/* $purple: #8048f3; */
+$highlight-purple: #9e6aff;
```

Comment:
Replaced this purple color with the new one from the designs (used to be #8950d9)